src/Stratis.Bitcoin.Features.Consensus/Rules/ConsensusRules.cs

```diff
@@ -134,4 +135,43 @@ public void SetErrorHandler(ConsensusRule errorHandler)
             }
         }
     }
+
+    /// <summary>
+    /// Consensus rules that represent a Proof-Of-Work blockchain.
+    /// </summary>
+    public class PowConsensusRules : ConsensusRules
+    {
+        /// <summary>
+        /// Initializes an instance of the object.
+        /// </summary>
+        public PowConsensusRules(Network network, ILoggerFactory loggerFactory, IDateTimeProvider dateTimeProvider, ConcurrentChain chain, NodeDeployments nodeDeployments, ConsensusSettings consensusSettings, ICheckpoints checkpoints)
+            : base(network, loggerFactory, dateTimeProvider, chain, nodeDeployments, consensusSettings, checkpoints)
+        {
+        }
+    }
+
+    /// <summary>
+    /// Consensus rules that represent a Proof-Of-Stake blockchain.
+    /// </summary>
+    /// <remarks>
+    /// A Proof-Of-Stake blockchain as implemented in this code base represents a hybrid POS/POW consensus model.
+    /// </remarks>
+    public class PosConsensusRules : ConsensusRules
+    {
+        /// <summary>Database of stake related data for the current blockchain.</summary>
+        public StakeChain StakeChain { get; }
```

Comment:
Readonly property?

Reply:
yes

Reply:
Ah actually this can't be read-only as its a public property (its already readonly as it a getter)

Reply:
Oh yes you are right 👍